src/olympia/devhub/tests/test_views.py

```diff
@@ -1492,7 +1491,7 @@ def test_lwt_submit_redirects_to_addon_submit(self):
         url = reverse('devhub.themes.submit')
         response = self.client.get(url, follow=True)
         self.assert3xx(
-            response, reverse('devhub.submit.distribution'), 301)
+            response, reverse('devhub.submit.distribution'), 302)
```

Comment:
The default for `django.shortcuts:redirect` changed, might need review of other redirects but that's not too dangerous imho. Especially in this case the redirect was related to a django-waffle so a 302 sounds even more appealing than 301.